src/v2/Apps/Partner/Routes/Overview/index.tsx

```diff
@@ -41,7 +41,7 @@ const Overview: React.FC<OverviewProps> = ({ partner }) => {
 
       <ShowsRailFragmentContainer mt={4} mb={[4, 80]} partner={partner} />
       {displayArtistsSection && (
-        <ArtistsRailFragmentContainer partner={partner} />
+        <ArtistsRailFragmentContainer mt={4} mb={[4, 80]} partner={partner} />
```

Comment:
the `[4, 80]` values feel odd to me-- what design is this trying to match?

Reply:
![image](https://user-images.githubusercontent.com/79979820/119947604-99be7300-bfa0-11eb-88a3-26b08c1fd3e3.png)

Reply:
https://www.figma.com/file/x78XGXMM1LhOdQY8TxkXkR/Partner-Profile?node-id=385%3A12025